### DISCUSSION TITLE
The breadcrumb navigation should be structured in <ol></ol>

### DISCUSSION THREAD

**Discussion by dominikkrulak** [saadeghi/daisyui#1703] [Ideas / feature requests]
It's simple. A breadcrumb navigation is a set of ordered links. Therefore I suggest using `ol`HTML  tag element in breadcrumb navigation.

```
<ol>
    <li><a href="">Link A</a></li>
    <li><a href="">Link B</a></li>
    <li><a href="">Link C</a></li>
</ol>
```

Currently if you use `ol` in breadcrumb navigation it won't style by default. Here is an example from [w3.org](https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/examples/breadcrumb/).

**Comment by saadeghi** [comment]
>Currently if you use `ol` in breadcrumb navigation it won't style by default.

Since daisyUI version [`2.51.3`](https://github.com/saadeghi/daisyui/compare/v2.51.2...v2.51.3) You can use `<ol>` as well.  
Example: https://play.tailwindcss.com/faMxKwMoS2
